Tools/px4airframes/srcparser.py

```diff
@@ -107,6 +107,8 @@ def GetImageName(self):
             return "Balloon"
         elif (self.type == "Vectored 6 DOF UUV"):
             return "Vectored6DofUUV"
+        elif (self.type == "Space Robot"):
```

Comment:
Why space robot? Why not spacecraft that is consistent with the vehicle type enum?

Reply:
Interesting! Will check this out